src/cdk/table/table.spec.ts

```diff
@@ -482,6 +482,19 @@ describe('CdkTable', () => {
     ]);
   });
 
+  it('should be able to nest tables', () => {
+    const thisFixture = createComponent(NestedHtmlTableApp);
+    thisFixture.detectChanges();
+    const outerTable = thisFixture.nativeElement.querySelector('table');
+    const innerTable = outerTable.querySelector('table');
+
+    expect(outerTable).toBeTruthy();
+    expect(outerTable.querySelector('tbody').rows.length).toBe(3);
```

Comment:
Would like to add tests regarding the cells as well, rather than just the row count. Can you test the table contents?

Reply:
I wanted to avoid testing the contents, because it gets weird with whitespace once you start nesting elements. I'll add assertions for the number of cells.